src/main/java/core/basesyntax/FruitShopApplication.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileRead;
+import core.basesyntax.service.FileWrite;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.FileReadImpl;
+import core.basesyntax.service.impl.FileWriteImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.FruitOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import core.basesyntax.validation.DataValidator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopApplication {
+    private static final Map<String, Integer> inventory = new HashMap<>();
```

Comment:
Static mutable state (the HashMap named 'inventory') could lead to difficult-to-diagnose bugs and concurrency issues. Consider encapsulating this within an instance of a class instead of making it static.